shopinvader/services/customer.py

```diff
@@ -68,37 +68,56 @@ def _validator_sign_in(self):
     def _validator_create(self):
         address = self.component(usage="addresses")
         schema = address._validator_create()
-        exclude_keys = ["type"]
-        for key in exclude_keys:
-            schema.pop(key, None)
         schema.update(
             {
-                "email": {"type": "string", "required": True},
-                "external_id": {"type": "string", "required": True},
-                "vat": {"type": "string", "required": False},
-                "company_name": {"type": "string", "required": False},
-                "function": {"type": "string", "required": False},
+                # Email is mandatory as of today
+                "email": {"type": "string", "required": True, "nullable": False},
```

Comment:
it's a breaking change. You can't break the existing API.

Reply:
I didn't change anything, email was already required, I just added a comment. BTW it couldn't be different because `partner_email` is required on s.partner :wink: